### ISSUE TITLE
Progress dot should have timeout and slow down over period of time.

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#1374]
Currently we don't have any timeout for progress dot and also when a operation gets longer than it should slow down over period of time. 

```
Disk size: 20 GB
-- Starting Minishift VM ..........................................................
............................................................................................
............................................................................................
............................................................................................
............................................................................................
```

**Comment by gbraad** [comment]
if `slowDown`, we use a `easeOffFactor` or `slowDownFactor`? Which does it based on a modulo or similar, so if 10 doys have been written we add an additional second to the interval.

**Comment by gbraad** [comment]
But I believe the biggest issue is that we can not ctrl+c out of the process. I am pretty sure it responded to a break before...

**Comment by praveenkumar** [comment]
@gbraad `ctrl+c` works in Linux need to check in windows.

```
$ ./minishift start 
-- Checking if KVM driver is installed ... 
   Driver is available at /usr/sbin/docker-machine-driver-kvm ... 
   Checking driver binary is executable ... OK
-- Starting local OpenShift cluster using 'kvm' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM .....^C
[prkumar@dhcp193-13 bin]$ 
```

**Comment by budhrg** [comment]
@gbraad , `ctrl+c` works in Windows too for me
```
PS C:\gowork\src\github.com\minishift\minishift> minishift start --iso-url file://C:/ms_cache/iso/1.1.0/minishift-b2d.iso
-- Checking if Hyper-V driver is configured ... OK
-- Starting local OpenShift cluster using 'hyperv' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM ................
```

**Comment by gbraad** [comment]
```
PS C:\work\src\github.com\minishift\minishift\out\windows-amd64> .\minishift.exe start
-- Checking if Hyper-V driver is configured ... OK
-- Starting local OpenShift cluster using 'hyperv' hypervisor ...
-- Starting Minishift VM ...........................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
....................................................................................................
..................................................
PS C:\work\src\github.com\minishift\minishift\out\windows-amd64> ^C
PS C:\work\src\github.com\minishift\minishift\out\windows-amd64> ^C
PS C:\work\src\github.com\minishift\minishift\out\windows-amd64> ^C
PS C:\work\src\github.com\minishift\minishift\out\windows-amd64> ^C
PS C:\work\src\github.com\minishift\minishift\out\windows-amd64> ^C
```

Not in my case... I need to kill `minishift.exe` from the process manager. You can see the break has been received by powershell...

**Comment by LalatenduMohanty** [comment]
@gbraad The issue is not reproducible right?

**Comment by LalatenduMohanty** [comment]
Fixed by #1383
